.github/actions/setup-job/action.yml

```diff
@@ -65,6 +72,12 @@ runs:
           exit 1
         fi
 
+    - name: Install rust target
+      working-directory: ${{ env.RUST_WORKSPACE }}
```

Comment:
Why do we need to change directory?

Reply:
we need to keep this otherwise rustup installs the target for the current stable and not for the version that is defined in discovery_engine_core/rust-toolchain.toml